eng/pipelines/runtimelab/runtimelab-post-build-steps.yml

```diff
@@ -26,13 +27,14 @@ steps:
 
   # Build coreclr native test output outside of official build
   - ${{ if ne(parameters.isOfficialBuild, true) }}:
-    - ${{ if and(eq(parameters.archType, 'wasm'), ne(parameters.nameSuffix, '')) }}:
+    - ${{ if and(eq(parameters.archType, 'wasm'), and(ne(parameters.nameSuffix, ''), eq(parameters.wasmEnableThreadsArg, ''))) }}:
```

Comment:
BTW, it turns out `and` is a vararg operator, e. g. `and(a, b, c)` is meant to work.